tests/test_quantumBackedButler.py

```diff
@@ -128,7 +135,9 @@ def test_initialize(self) -> None:
         """Test for initialize factory method"""
 
         quantum = self.make_quantum()
-        qbb = QuantumBackedButler.initialize(config=self.config, quantum=quantum, dimensions=self.universe)
+        qbb = QuantumBackedButler.initialize(
```

Comment:
Do any of these tests fail without the dataset types? I assume without it all falls back to the graph dataset type which will mostly work fine.

Reply:
We do not have tests here which override storage classes, so in this case it's not strictly necessary to pass that parameter here, tests will work OK even without it.